### DISCUSSION TITLE
Question about UNITs in OpenRadioss

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1956] [Q&A]
Hello everyone,

May I ask I question here about the system units? Can I use US. Unit (inch) in stead of SI Unit? like PSI, inch and s? and mass with lbf s^2/inch. what is the format should be and any example used US unit?

Thanks for your comments

Best
Lee

**Comment by MarianBulla** [comment]
Hello Lee,
basically, OpenRadioss does not make usage of any units. The definition of the unit systems in the begin card is used for recomputing values, if different unit systemes are used within one model file. (There are only few exceptions, e.g. the GAS material and the pre-defined material law.)
If you add just 1.0 for all values (mass, length and time at both lines in the /BEGIN card), then the recomputation will not change anything with respect to units.
Then, you are free to use whatever units you want, including US units or anthing else.
Just take care on the consistency of all your values.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks so much for your comments/help. If I am understanding correctly as long as I have constant unit in my material cards the system should calculate the units of other quantities automatically. What I was given in the /begin card shouldn't be matter and I can give just 1.0 for all mass, length and time right?

One more question for you regarding the part contains different type of elements. I have I a part (not an assembly) contains solid and shell elements simultaneously, how can I define the /Part cards? It will require both solid/shell section definition and material Law. In the /part card, I can only include one type of element and its section definition. Thanks again for all your answers and help. There are so helpful.

Regards
Lee

**Reply by MarianBulla** [reply]
Yes, exactly. You are responsible for all the values. 

Regarding the /PART and /PROP definition: One part can contain only one kind/type of elements.
If you have shells and solids, they must be separated into different parts with different properties. 
But can have assigned the same material (if it is compatible with both types of elements).

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks again. Regrading the /Part and /Prop definition. Please see attached screenshot. In my application, the center hub was modeled as solid and the blades were modeled as shell. There are some nodes shared at the root edges of the blades and the center hub. As you said one part can only contain one type of element. So in my situation, I should use some kind of tie constraint to connect the two parts? Thanks for your time
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/0b8961d3-ca0b-4409-a1f6-f86b2e2a04b8)

**Reply by PaulAltair** [reply]
Yes and no... it is fine for the shared nodes to be 'in' both parts from that point of view, however, since the solids have no rotational degree of freedom stiffness, it would actually be better practice to use a tied contact to connect them

**Reply by li2qn** [reply]
Thanks so much. That helps.
